mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +35,66 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Enlarge Image
+   */
+  isZoomable?: boolean;
+
+  /**
+   * Large Image
+   */
+  isZoomed?: boolean;
+
+  /**
+   * Open Image
+   */
+  onOpenChange?: (arg: boolean) => void;
 }
 
 /**
- * Сomponent for displaying images
+ * Component for displaying images
  */
 export const Image = (props: ImageProps) => {
-  const className = clsx(styles.image, props.className);
+  const imageClass = clsx(styles.image, props.className);
+  const [isOpen, setIsOpen] = useState(false);
 
-  return (
+  /**
+   * Test
+   */
+  const handleOpenChange = (value: boolean) => {
+    setIsOpen(value);
+    if (props.onOpenChange) {
+      props.onOpenChange(value);
+    }
+  };
+  const imageElement = (
     <img
       src={props.src}
       alt={props.alt}
-      className={className}
+      className={imageClass}
       data-cy={props.dataCy}
     />
   );
-};
 
+  return (
+    props.isZoomable
+      ? (
+        <DialogRoot
+          open={props.isZoomed ?? isOpen}
+          onOpenChange={handleOpenChange}
+        >
+          <ModalTrigger>
```

Comment:
Why do you need it? We have alreaady modal component

Reply:
<ModalTrigger>
      {imageElement}
      <ModalContent>
        <img src={props.src} alt={props.alt} className={styles.zoomedImage} />
      </ModalContent>
    </ModalTrigger>
    
    Use ModalTrigger and ModalContent ? Without Modal cannot use useState